rq/group.py

```diff
@@ -1,38 +1,40 @@
-from typing import List, Optional
+from typing import List, Optional, Union
 from uuid import uuid4
 
 from redis import Redis
 from redis.client import Pipeline
 
-from .exceptions import NoSuchBatchError
+from . import Queue
+from .exceptions import NoSuchGroupError
 from .job import Job
+from .queue import EnqueueData
 from .utils import as_text
 
 
-class Batch:
-    """A Batch is a container for tracking multiple jobs with a single identifier."""
+class Group:
+    """A Group is a container for tracking multiple jobs with a single identifier."""
 
-    REDIS_BATCH_NAME_PREFIX = 'rq:batch:'
-    REDIS_BATCH_KEY = 'rq:batches'
+    REDIS_BATCH_NAME_PREFIX = 'rq:group:'
```

Comment:
This should be `REDIS_GROUP_NAME_PREFIX` and `REDIS_GROUP_KEY`

Reply:
Done, thank you